packages/e2e-tests/puppeteer/__tests__/optional/seed-download-cancel.test.ts

```diff
@@ -110,13 +110,12 @@ describe('Optional Integration Tests', () => {
     // TODO: Verify withdrawal for direct funding once it's implemented
     // see https://github.com/statechannels/monorepo/issues/1546
 
-    console.log('Wait for Wallet to be hidden');
-    // Ensure the wallet is not visible
-    await forEachTab(waitForWalletToBeHidden);
-
     console.log('Wait for the "Restart Download" button to appear');
     await waitForFinishedOrCanceledDownload(web3tTabB);
 
+    console.log('Wait for the ChannelUpdated "closed" state');
+    await forEachTab(tab => waitForClosedState(tab));
+
```

Comment:
The logic for this: `await waitForFinishedOrCanceledDownload(web3tTabB);` is that the button should appear when the channel is closed, and the torrent is deleted. The torrent is deleted once the closeChannel call is finished. So I believe that the `    await forEachTab(tab => waitForClosedState(tab));` it's not necessary.

Reply:
Will let you + @kerzhner decide and edit this branch as needed. When I ran it locally I observed the channel _not_ closing.